internal/storage/v2/elasticsearch/tracestore/to_dbmodel_test.go

```diff
@@ -336,6 +339,59 @@ func TestToDBModel(t *testing.T) {
 	}
 }
 
+func TestToDbModel_Fixtures(t *testing.T) {
+	span := getDbSpan(t)
+	td, err := FromDBModel([]dbmodel.Span{span})
+	require.NoError(t, err)
+	assertTraces(t, td)
+	spans := ToDBModel(td)
+	assertSpans(t, spans)
+}
+
+func writeActualData(t *testing.T, name string, data []byte) {
+	var prettyJson bytes.Buffer
+	err := json.Indent(&prettyJson, data, "", "  ")
+	require.NoError(t, err)
+	err = os.WriteFile("actual_"+name+".json", prettyJson.Bytes(), 0o644)
+	require.NoError(t, err)
+}
+
+func assertTraces(t *testing.T, td ptrace.Traces) {
+	if !assert.Equal(t, getPTraces(t), td) {
```

Comment:
Assert function should be passed expected/actual values, it should not be loading one of those itself.